tools/release_do_nothing.py

```diff
@@ -0,0 +1,758 @@
+        f"https://github.com/SciTools/iris/releases/tag/{release_strings.tag}\n"
+        "You have confirmed that Read the Docs, PyPI and conda-forge have all "
+        "updated correctly. Include the following links in the release "
+        "notes:\n\n"
+        f"https://scitools-iris.readthedocs.io/en/{release_strings.tag}/\n"
+        f"https://pypi.org/project/scitools-iris/{release_strings.release}/\n"
+        f"https://anaconda.org/conda-forge/iris?version={release_strings.release}\n"
+    )
+    _wait_for_done(message)
+
+    message = (
+        "Update the release page in GitHub discussions, with the above links "
+        "and anything else appropriate.\n"
+        "https://github.com/SciTools/iris/discussions"
+    )
+    _wait_for_done(message)
+
+
+def twitter_announce(
+    release_strings: ReleaseStrings, first_in_series: bool
+) -> None:
+    message = (
+        "Announce the release via https://twitter.com/scitools_iris, and any "
+        "other appropriate message boards.\n"
+        "Any content used for the announcement should be stored in the "
+        "SciTools/twitter-scitools-iris GitHub repo.\n"
+    )
+    if not first_in_series:
+        message += (
+            f"Consider replying within an existing {release_strings.series} "
+            "announcement thread, if appropriate."
+        )
+    _wait_for_done(message)
+
+
+def update_citation(
+    release_strings: ReleaseStrings, is_release_candidate: bool
+) -> None:
+    if not is_release_candidate:
+        src_dir = Path(__file__).parents[1] / "docs" / "src"
+        citation_rst = src_dir / "userguide" / "citation.rst"
+        assert citation_rst.is_file()
+        message = (
+            f"Follow the Pull Request process to update {citation_rst.name} "
+            "with the correct dates, DOI and version string for "
+            f"{release_strings.tag}.\n"
+            f"{citation_rst.absolute()}\n\n"
+            f"The PR should target {release_strings.branch} (prior to merge-back)."
+        )
+        _wait_for_done(message)
+
+
+def merge_back(
+    release_strings: ReleaseStrings, first_in_series: bool, rsts: WhatsNewRsts
+) -> None:
+    _break_print("Branch merge-back ...")
+
+    merge_commit = (
+        "BE SURE TO MERGE VIA A MERGE-COMMIT (not a squash-commit), to "
+        "preserve the commit SHA's."
+    )
+
+    if first_in_series:
+        # TODO: automate
+
+        working_branch = release_strings.branch + ".mergeback"
+        _delete_local_branch(working_branch)
+        message = (
+            "Checkout a local branch from the official ``main`` branch.\n"
+            "git fetch upstream;\n"
+            f"git checkout upstream/main -b {working_branch};"
+        )
+        _wait_for_done(message)
+
+        message = (
+            f"Merge in the commits from {release_strings.branch}.\n"
+            f"{merge_commit}\n"
+            f"git merge upstream/{release_strings.branch} --no-ff;"
```

Comment:
add in a -m "merging release branch to main"

Reply:
e3d237c18